src/bin/lfortran.cpp

```diff
@@ -2664,6 +2664,7 @@ int main_app(int argc, char *argv[]) {
             tmp_o = arg_file;
         }
         if (err && !compiler_options.continue_compilation) return err;
+        if (err) tmp_o = "";
         err_ = err;
         object_files.push_back(tmp_o);
```

Comment:
I see, this modifies a lot of tests.

How about the following:

```suggestion
        err_ = err;
        if (err != 0) object_files.push_back(tmp_o);
```

Also, do you know what is the use `err_`. If it corrects all the error code, then here it needs to be 
```
err_ += err;
```
right?

Reply:
The value `err_` is 0 or 1 according to the absence or presence of error.